### ISSUE TITLE
wayvnc incorrectly negotiates tight encoding when resolution is too large for it

### ISSUE THREAD

**Issue by colemickens** [any1/wayvnc#46]
See: https://github.com/TigerVNC/tigervnc/issues/1054#event-3505527978

It appears the server might not be following spec in regards to advertising encodings available for a given remote resolution.

**Comment by any1** [comment]
Yeah, this is known. I just haven't gotten around to fixing it yet.
